frontend/src/libs/components/input/input.tsx

```diff
@@ -70,9 +71,7 @@ const Input = <T extends FieldValues>({
         disabled={isDisabled}
         inputMode={inputMode}
       />
-      <span className={styles['error-message']}>
-        {hasError && (error as string)}
-      </span>
+      <ErrorMessage error={error as string} />
```

Comment:
Have you removed in `styles` for `input` this `error-message` styling?

Reply:
agreed